server_tests/test/clientTests.js

```diff
@@ -1086,25 +1067,25 @@ describe("RUST API TEST SUITE", function() {
         it("Expect success. /osc/commands/status successfully grabs command status after take picture has been called", function() {
             this.timeout(timeoutValue);
             return testClient.takePicture(sessionId, function(res) {
-                Comparison.oscCommandsStatusOutput(res, {'name': 'camera.takePicture', 'id': res.body.id});
+                validate.done(res.body, schema.names.commandsStatus);
```

Comment:
This is actually the status callback, not completion. It means that this will fail, and should instead be `validate.inProgress(...)`. However, without deferred promises, an error here will be lost and thrown into the void.

(also a `inProgress` helper function needs to be added to validator.js)